docs/cli/oc4ids.rst

```diff
@@ -0,0 +1,123 @@
+OC4IDS Commands
+================
+
+The following commands may be used when working with `Open Contracting for Infrastructure <https://standard.open-contracting.org/infrastructure>`__ data.
+
+oc4ids-transforms
+-----------------
+
+This command takes a list of OCDS releases and transforms them into a single OC4IDS project.
+
+`The logic for the mappings between OCDS and OC4IDS fields is documented here <https://standard.open-contracting.org/infrastructure/latest/en/cost/#mapping-to-ids-and-from-ocds>`__.
+
+Optional arguments:
+
+* ``--id ID`` the Project ID of the created project
+* ``--package`` wrap the project in a package
+* ``--options OPTIONS`` a comma seperated list of optional tranforms
+* ``--all`` run all optional transforms
+
+Arguments that can be applied if the project is in a package:
+
+* ``--uri URI`` if ``--package`` is set, set the record package's ``uri`` to this value
+* ``--published-date PUBLISHED_DATE`` if ``--package`` is set, set the record package's ``publishedDate`` to this value
+* ``--publisher-name PUBLISHER_NAME`` if ``--package`` is set, set the record package's ``publisher``'s ``name`` to this value
+* ``--publisher-uri PUBLISHER_URI`` if ``--package`` is set, set the record package's ``publisher``'s ``uri`` to this value
+* ``--publisher-scheme PUBLISHER_SCHEME`` if ``--package`` is set, set the record package's ``publisher``'s ``scheme`` to this value
+* ``--publisher-uid PUBLISHER_UID`` if ``--package`` is set, set the record package's ``publisher``'s ``uid`` to this value
+* ``--fake`` if ``--package`` is set, set the record package's required metadata to dummy values
+
+Transforms
+~~~~~~~~~~
+
+The transforms that are run are described here.
+
+* ``additional_classifications``, ``description``, ``sector``, ``title``: populate top-level fields with their equivalents from ``planning.project`` 
```

Comment:
@duncandewhurst Have you checked that this part of the documentation is correct?

If these can't be turned on/off, is it helpful to the user to list them, or is it largely duplicating of the mapping in the OC4IDS docs?